projects/core/src/lib/users/user.service.ts

```diff
@@ -190,49 +195,55 @@ export class UserService {
      */
     login(credentials: {email: string; password: string}): Observable<Session> {
         const result = new ReplaySubject<Session>();
-        this.backend
-            .loginUser(credentials)
-            .pipe(mergeMap((response) => this.createSession(response)))
-            .subscribe(
-                (session) => {
-                    this.session$.next(session);
-                    result.next(session);
+
+        new SessionApi(new Configuration({basePath: this.config.API_URL}))
+            .loginHandler({
+                userCredentials: {
```

Comment:
can you not just pass the credentials without unpacking?

Reply:
[794cdc0](https://github.com/geo-engine/geoengine-ui/pull/687/commits/794cdc015b8891f366415fcd572c528d8d734f82)